rust/cw-contracts/cw-address-list/Makefile

```diff
@@ -0,0 +1,54 @@
+ADMIN_TESTNET=testnet-adm
```

Comment:
we should remove this makefile and use scripts that are common to all contracts IMO

Reply:
Done